Header header #Header for message type
int32 type_sender
int32 type_command_filter
bool mecanum_movement_mode #
bool arm_velocity_mode #true == arm commands are velocities else positions
bool arm_angle_reset
float64 robot_x #robot front movement Basic movement commands
float64 robot_y #robot side movement
float64 robot_zrot #robot z rotation
float64 arm_front_left #front arm angle or up/down movement 
float64 arm_front_right #back arm angle or up/down movement
float64 arm_back_left #front arm angle or up/down movement 
float64 arm_back_right #back arm angle or up/down movement


